archivesunleashed/auk
# Highlight Notebooks in Documentation and Collection View

**GitHub issue(s)**:

If you are responding to an issue, please mention their numbers below.

* #285

# What does this Pull Request do?

This pull request adds in discussion of the notebooks as a "prototype" service in the documentation and FAQ, as well as in the collection view. The first two are straightforward, I am less sure about the latter. 

Here is what the collection view looks like in this PR:

![Screen Shot 2019-04-17 at 12 09 30 PM](https://user-images.githubusercontent.com/3834704/56303318-ab6a9680-6109-11e9-8a73-14c698127dd9.png)

Note that I've added an additional link to the documentation section under the notebook.

# How should this be tested?

TravisCI should turn green, but more importantly, reviewers should look at the additional prose and make sure it is clear as well as the look and feel of the link. I am looking for any suggestions around the best way to highlight this without cluttering the page..

# Interested parties

@ruebot @greebie @SamFritz
